h/oauth/errors.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from oauthlib.oauth2 import InvalidGrantError
+
+
+class MissingJWTGrantTokenClaimError(InvalidGrantError):
+    def __init__(self, claim, claim_description=None):
+        if claim_description:
+            self.description = 'Missing grant token {} ({}).'.format(claim_description, claim)
```

Comment:
Yep, I agree. I would like to leave the sentence structure though because it's how [oauthlib](https://github.com/idan/oauthlib/blob/master/oauthlib/oauth2/rfc6749/errors.py#L137) [error](https://github.com/idan/oauthlib/blob/master/oauthlib/oauth2/rfc6749/errors.py#L153) [messages](https://github.com/idan/oauthlib/blob/master/oauthlib/oauth2/rfc6749/errors.py#L149) [look](https://github.com/idan/oauthlib/blob/master/oauthlib/oauth2/rfc6749/errors.py#L145) [like](https://github.com/idan/oauthlib/blob/master/oauthlib/oauth2/rfc6749/errors.py#L141).

But I'll change it to `Missing claim 'iss' (issuer) from grant token`.